telegraf/manifests/deis-monitor-telegraf-daemon.yaml

```diff
@@ -41,6 +41,13 @@ spec:
             value: "metrics"
           - name: "NSQ_ENDPOINTS"
             value: http://$(DEIS_NSQD_SERVICE_HOST):$(DEIS_NSQD_SERVICE_PORT_HTTP)
+          - name: DEIS_LOGGER_REDIS_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: logger-redis-creds
+                key: password
+          - name: "REDIS_SERVERS"
+            value: tcp://:$(DEIS_LOGGER_REDIS_PASSWORD)@$(DEIS_LOGGER_REDIS_SERVICE_HOST):$(DEIS_LOGGER_REDIS_SERVICE_PORT)
```

Comment:
IMO this is ok for now, but @jchauncey how do you feel about doing this in a future PR:

- store the password in a secret
- making this value a go template that looks like `tcp://{{.DeisLoggerRedisPassword}}#$(DEIS_LOGGER_REDIS_SERVICE_HOST):$(DEIS_LOGGER_REDIS_SERVICE_PORT)`
- Building the final value of `REDIS_SERVERS` by executing the aforementioned template with the value of the secret assigned to `DeisLoggerRedisPassword` in the template